src/main/kotlin/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,16 @@
+package calculator
+
+class StringCalculator(private val input: Input) {
+
+    fun calculate(): Number {
+        val expression = Expression(input)
+        return expression.calculate()
+    }
+}
+
+fun main(args: Array<String>) {
```

Comment:
```suggestion
fun main() {
```
kotlin에서는 args 를 추가하지 않아도 됩니다!